file/writable_file_writer.cc

```diff
@@ -470,6 +470,11 @@ IOStatus WritableFileWriter::WriteBuffered(const char* data, size_t size) {
       allowed = left;
     }
 
+#ifndef NDEBUG
+    size_t* call_back_para[] = {&allowed, &size};
```

Comment:
I think the internal linter will recommend using `std::array` instead of raw array, let's just make the linter happy given the extra effort is trivial.

Reply:
I just remove this call_back_para. 
There is a simple way to reproduce a WAL corruption by doing concurrent write and FlushWAL.  It is not necessary to reproduce another WAL corruption issue with a complicated  code.